js/data.js

```diff
@@ -37,4 +43,40 @@ const maxLng = 139.80000;
 
 const QUANTITYADS = 10;
 
-export {TITLES, TYPES, TIMES, FEATURES, DESCRIPTIONS, PHOTOLINKS, QUANTITYADS, minLat, maxLat, minLng, maxLng};
+// out add
+
+const createOutAdd = () => {
+  const location = {
+    lat: getFloatingPointNumber(minLat, maxLat, 5),
+    lng: getFloatingPointNumber(minLng, maxLng, 5),
+  };
+
+  const author = {
+    avatar: `img/avatars/user${getNumberAvatar()}.png`,
+  };
+
+  const offer = {
+    title: TITLES[getRandomNumber(0, TITLES.length - 1)],
+    adress: `${location.lat}.lat, ${location.lng}.lng`,
+    price: getRandomNumber(1000, 40000),
+    type: TYPES[getRandomNumber(0, TYPES.length - 1)],
+    rooms: getRandomNumber(1, 5),
+    guests: getRandomNumber(1, 10),
+    checkin: TIMES[getRandomNumber(0, TIMES.length -1)],
+    checkout: TIMES[getRandomNumber(0, TIMES.length - 1)],
+    features: getRandomFeatures(),
+    description: DESCRIPTIONS[getRandomNumber(0, DESCRIPTIONS.length - 1)],
+    photos: getRandomPhotos(),
+  };
+
+  return {
+    author,
+    offer,
+    location,
+  };
+};
+
+const createSimilarAdd = () => Array.from({length: QUANTITYADS}, createOutAdd);
```

Comment:
```suggestion
const createSimilarAdd = (count) => Array.from({length: count}, createOutAdd);
```
вот так вообще красота будет)